low-level/framework/platforms/node/sas_interface.py

```diff
@@ -0,0 +1,152 @@
+# CORTX Python common library.
```

Comment:
Put this under
low-level/framework/platforms/server/sas.py

Reply:
addressed 👍 
What should be the Name for the Error class?? I have kept it still as `InterfaceError`, another candidate in my mind is `PhyInterfaceError` with `PhyInterface` as a superclass for `SAS` and `Network` classes.